src/Tokenizers/Tokenizer.php

```diff
@@ -642,21 +642,10 @@ public function replaceTabsInToken(&$token, $prefix=' ', $padding=' ', $tabWidth
                 $tabNum++;
 
                 // Move the pointer to the next tab stop.
-                if (($currColumn % $tabWidth) === 0) {
-                    // This is the first tab, and we are already at a
-                    // tab stop, so this tab counts as a single space.
-                    $currColumn++;
-                } else {
-                    $currColumn++;
-                    while (($currColumn % $tabWidth) !== 0) {
-                        $currColumn++;
-                    }
-
-                    $currColumn++;
-                }
-
-                $length     += ($currColumn - $lastCurrColumn);
```

Comment:
`$lastCurrColumn` appears to be unused now and the declaration above can be removed